src/Nox.Types.EntityFramework/Types/Url/UrlConverter.cs

```diff
@@ -2,8 +2,7 @@
 
 namespace Nox.Types.EntityFramework.Types;
 
-public class UrlConverter : ValueConverter<Url, string>
+public class UrlConverter : ValueConverter<Url, System.Uri>
```

Comment:
Let's switch back to string as the underlying type, because it's safer to use across different database providers.

Reply:
Switched back to string.